addons/l10n_eg_edi_eta/views/product_template_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="product_template_only_form_view_inherit_l10n_eg_eta_edi" model="ir.ui.view">
+            <field name="name">product.template.form.l10n_eg_eta_edi</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_only_form_view"/>
+            <field name="arch" type="xml">
+                <page name="invoicing" position="inside">
+                    <group name="l10n_eg_eta_edi" string="Egyption Electronic Invoicing" attrs="{'invisible': [('product_variant_count', '>', 1)]}">
+                        <field name="l10n_eg_eta_code" help="ETA Field for GS1/EGS product codes. Please use the barcode field to store GS1/EGS ETA code if possible"/>
```

Comment:
Maybe it should say: if you want to use it as a real barcode, please use the barcode instead, because this field can not be scanned.  
We need to pass GS1 or EGS in the XML, but sometimes their barcodes are internal ones.